int64 v_data
int64 a_data
int64 time_data
